backend/src/vantracking/factory.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+from src.vantracking.cachetools.cache import CachetoolsVanStateCache
+from src.vantracking.memcached.cache import MemcachedVanStateCache
+from src.vantracking.tracker import VanTracker
+
+
+def van_tracker() -> VanTracker:
+    """
+    Create a new van manager from the environment variable configuration. This will raise a ValueError if the
+    configuration is invalid.
+    """
+
+    config = {}
+    for key in os.environ:
+        if key.startswith("CACHE_"):
+            config[key[6:].lower()] = os.environ[key]
```

Comment:
This looks like a potentially unsafe access of the config dict, we should probably handle if that doesn't exist

Reply:
Not sure what you mean by this. It's a setter, so if the key doesn't exist it will just be populated with a value.

Reply:
Ahhh I'm dumb, I didn't read. It's fine then